utils.py

```diff
@@ -29,4 +29,7 @@ def color_to_hex(color) -> str:
     g = round(color['green'] * 255) if 'green' in color else 0
     b = round(color['blue'] * 255) if 'blue' in color else 0
 
-    return f'{r:02x}{g:02x}{b:02x}'
\ No newline at end of file
+    return f'{r:02x}{g:02x}{b:02x}'
+
+def is_white(color) -> bool:
+    return 'red' in color and color['red'] == 1 and 'blue' in color and  color['green'] == 1 and 'blue' in color and color['blue'] == 1
```

Comment:
```python
 return 'red' in color and color['red'] == 1 and 'green' in color and  color['green'] == 1 and 'blue' in color and color['blue'] == 1
 ```

Reply:
Used to fix the issue where a cell in google sheets will always have a color value if you assign one once even if you reset the color.

Reply:
Maybe I'm missing something but how does checking for `'blue' in color` twice help get over this issue?

Reply:
Oops typo, fixed